### PR TITLE
Add aria labels & accessibility fixes

### PR DESCRIPTION
**Pull request by gracez72**
Tickets: [TASK-34231](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/Adaptive%20Cards/Intern%20GitHub/Y20-VW/10?workitem=34231), [TASK-34217](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/Adaptive%20Cards/Intern%20GitHub/Y20-VW/10?workitem=34217), [TASK-34219](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/Adaptive%20Cards/Intern%20GitHub/Y20-VW/10?workitem=34219)

This PR adds aria labels for images and search boxes and addresses the errors found by the web parsing tool.

### CODE REVIEW

File: private-templates-service/client/src/components/Dashboard/RecentlyViewed/styled.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const RecentlyViewedBody = styled.div`
   justify-content: flex-start;
 `;
 
-export const RecentlyViewedItem = styled.h6`
+export const RecentlyViewedItem = styled.div`
```
Comment (beervoley) [inline private-templates-service/client/src/components/Dashboard/RecentlyViewed/styled.tsx]: this really didn't make a difference in the font-size?))

File: private-templates-service/client/src/components/NavBar/SearchBar/styled.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import styled from "styled-components";
 import { SearchBox } from "office-ui-fabric-react";
 import { BREAK, COLORS } from "../../../globalStyles";
+import * as STRINGS from "../../../assets/strings";
 
-export const StyledSearchBox = styled(SearchBox)`
+export const StyledSearchBox = styled(SearchBox).attrs({"ariaLabel": STRINGS.SEARCHBAR_DESCRIPTION})`
```
Comment (beervoley) [inline private-templates-service/client/src/components/NavBar/SearchBar/styled.tsx]: Kody asked to move the ariaLabel from **syled.tsx** to its component in the actual code. **<StyledSearchBox ariaLabel="..">**

File: private-templates-service/client/src/components/NavBar/styled.tsx [hunk 3]
```diff
@@ -37,7 +38,7 @@ export const BaselineBanner = styled.div`
   align-items: baseline;
 `;
 
-export const StyledLogo = styled.img`
+export const StyledLogo = styled.img.attrs({"aria-label": STRINGS.LOGO_DESCRIPTION})`
```
Comment (beervoley) [inline private-templates-service/client/src/components/NavBar/styled.tsx]: same comment as above

File: private-templates-service/client/src/components/SideBar/styled.tsx [hunk 4]
```diff
@@ -36,7 +37,7 @@ export const LogoWrapper = styled.div`
   padding-right: 48px;
 `;
 
-export const Logo = styled.img`
+export const Logo = styled.img.attrs({"aria-label": STRINGS.LOGO_DESCRIPTION})`
```
Comment (beervoley) [inline private-templates-service/client/src/components/SideBar/styled.tsx]: same comment as above

File: private-templates-service/client/src/components/Dashboard/RecentlyViewed/styled.tsx (hunk 1 above)
Comment (gracez72) [reply private-templates-service/client/src/components/Dashboard/RecentlyViewed/styled.tsx]: Nope, it stayed the same

File: private-templates-service/client/src/components/NavBar/SearchBar/styled.tsx (hunk 2 above)
Comment (gracez72) [reply private-templates-service/client/src/components/NavBar/SearchBar/styled.tsx]: Fixed!

File: private-templates-service/client/src/components/NavBar/NavBar.tsx [hunk 5]
```diff
@@ -55,7 +55,7 @@ const NavBar = (props: NavBarProps) => {
     return (
       <Banner>
         <MobileBanner>
-          <StyledLogo src={Logo} />
+          <StyledLogo aria-label={STRINGS.LOGO_DESCRIPTION} src={Logo} />
```
Comment (beervoley) [inline private-templates-service/client/src/components/NavBar/NavBar.tsx]: Please correct me if I’m mistaken but shouldn’t it be **ariaLabel**? aria-label would be correct in CSS

Comment (gracez72) [reply private-templates-service/client/src/components/NavBar/NavBar.tsx]: ariaLabel gives me an error, I double checked that aria-label sets the actual label in the accessibility tab

**Pull request merged**